tests/Humbug/Test/Phpunit/Logger/JsonLoggerTest.php

```diff
@@ -22,9 +25,8 @@ public function testShouldThrowExceptionWhenTargetIsNotSpecified()
 
     public function testShouldWriteLogsDuringDestruct()
     {
-        $jsonLogger = $this->getMock('\Humbug\Phpunit\Logger\JsonLogger', ['write'], ['/target/path']);
-
-        $jsonLogger->expects($this->once())->method('write');
+        $jsonLogger = m::mock("\\Humbug\\Phpunit\\Logger\\JsonLogger")->makePartial();
```

Comment:
why not using single quotes which also makes the expression more readable as well (well to be fair I'm more a single quote by default guy but if that's the default here nvm)